domain-service/credit-application/src/main/java/at/deloittedigital/core_banking/credit/api/TestController.java

```diff
@@ -0,0 +1,34 @@
+package at.deloittedigital.core_banking.credit.api;
+
+import at.deloittedigital.core_banking.credit.api.data.TestClientResponse;
+import at.deloittedigital.core_banking.credit.api.data.error.NotFoundException;
+import at.deloittedigital.core_banking.credit.domain.CreditApplicationService;
+import at.deloittedigital.core_banking.credit.domain.data.ClientInfo;
+import java.util.Optional;
+import javax.validation.Valid;
+import javax.validation.constraints.NotBlank;
+import org.springframework.validation.annotation.Validated;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.RequestParam;
+import org.springframework.web.bind.annotation.RestController;
+
+@RestController
+@Validated
+public class TestController {
+
+    private final CreditApplicationService creditApplicationService;
+
+    public TestController(CreditApplicationService creditApplicationService) {
```

Comment:
We could remove this constructor and use `@RequiredArgsConstructor` from lombok

Reply:
👍